sops/node-specific/GDI-SOP0008_node-helpdesk-ticket-classification.md

```diff
@@ -0,0 +1,506 @@
+
+2. [Glossary](#2-glossary)
+
+3. [Roles and Responsibilities](#3-roles-and-responsibilities)
+
+4. [Purpose](#4-purpose)
+
+5. [Scope](#5-scope)
+
+6. [Introduction and Background Information](#6-introduction-and-background-information)
+
+7. [Summary or Context Diagram](#7-summary-or-context-diagram)
+
+8. [Procedure](#8-procedure)
+
+9. [References](#9-references)
+
+### 1. Document History
+
+| Template Version | Instance version | Author(s) | Description of changes | Date |
+|---|---|---|---|---|
+|  ``v1``  |  | Elisavet Torstensson, Mattias Strömberg, Silvia Bahena| Created the first version of the SOP | 2025.02.04 |
+
+### 2. Glossary
+
+Find GDI SOPs common Glossary at the [charter document](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_charter.md).
+
+| Abbreviation |  Description |
+|---|---|
+| AF | Application Form|
+| API | Application Programming Interface |
+| DAA | Data Access Agreement|
+| DAC | Data Access Committee|
+| EBI | European Bioinformatics Institute |
+| EGA | European Genome-phenome Archive |
+| EMBL | European Molecular Biology Laboratory |
+| FTP | File Transfer Protocol |
+| GDI | Genomic Data Infrastructure |
+| HD | Helpdesk |
+| ID | Identity Document |
+| ISM | Information Service Management |
+| JSON | JavaScript Object Notation |
+| NBIS | National Bioinformatics Infrastructure Sweden |
+| NHD | Node Helpdesk |
+| OK | All correct |
+| ORR | Organisational Roles and Responsibilities |
+| RI | Research Infrastructure |
+| SOP | Standard Operating Procedure |
+| SP | Submitter Portal |
+| UU | University of Uppsala |
+| UX | User experience |
+| VHD | Virtual Helpdesk  |
+| XML | Extensible Markup Language  |
+
+| Term | Definition |
+|---|---|
+| API | Application Programming Interface is a connection between computers or between computer programs. It is a type of software interface, offering a service to other pieces of software. |
+| JIRA | is a software product developed by Atlassian that allows bug tracking, issue tracking and agile project management. |
+| JSON | JavaScript Object Notation is an open standard file format and data interchange format that uses human-readable text to store and transmit data objects consisting of name–value pairs and arrays (or other serializable values).|
+| Node Helpdesk | The support team responsible for handling tickets related to data management issues associated with the node in the GDI Project. |
+| Ticket | A ticket is a record in a ticket management system that tracks an issue, request, or task from creation to resolution. It helps streamline communication and workflows within teams or organizations.|
+| Ticket Classification | The process of categorising support requests based on their nature, so they can be assigned to the appropriate individual or team for resolution.  |
+| Virtual Helpdesk  | The initial point of contact for inquiries and requests related to data management in the GDI project. |
```

Comment:
I just noticed that these definitions are different than the ones in the Charter. I would say that the easiest and most consistent is to make them the same.